api/RestfulService.php

```diff
@@ -67,6 +67,22 @@ public static function set_default_curl_options($optionArray) {
 	}
 
 	/**
+	 * Parses cURL options defined in YML configuration.
+	 *
+	 * @return array
+	 */
+	public static function get_default_curl_options() {
+		$rawOptions = Config::inst()->get('RestfulService', 'default_curl_options');
```

Comment:
On iPad at moment, but if I recall, the other methods nearby use this same method. It may be more significant of a change but I'd be happy to do that in the other methods as well (so long as everyone is ok with that update and the impact it would have, if any, on other branches this would merge into).  If this doesn't become a part of `3` (I suspect it does) we would expect any other additional methods to be tweaked as well, I suppose 

Is this common elsewhere in the code base? I canot check at the moment.